homeassistant/components/roborock/entity.py

```diff
@@ -57,16 +57,13 @@ def get_cache(self, attribute: CacheableAttribute) -> AttributeCache:
         """Get an item from the api cache."""
         return self._api.cache[attribute]
 
-    async def send(
+    async def _send_command(
```

Comment:
Given this does not use `self` it can be a method outside of this class, or a `@classmethod`